src/chi/components/modal/modal.scss

```diff
@@ -58,23 +58,27 @@ $animation-duration: 0.5s;
         cursor: pointer;
         display: flex;
         left: 1rem;
+        opacity: .6;
         outline: none;
         padding: 0;
         position: absolute;
         top: 1.25rem;
       }
 
       &::before {
-        background-image: url(icon-back.svg);
-        content: '';
+        content: '\e91e';
```

Comment:
This is not the icon for icon-chevron-left. Please replace with `\e91f`

Reply:
done